lib/daru/dataframe.rb

```diff
@@ -1346,7 +1346,7 @@ def index= idx
     #   df.vectors = Daru::Index.new([:foo, :bar, :baz])
     #   df.vectors.to_a #=> [:foo, :bar, :baz]
     def vectors= idx
-      unless index.is_a?(Daru::Index)
+      unless idx.is_a?(Daru::Index)
```

Comment:
The original version is correct. Its checking if the argument `idx` is not of type `Daru::Index`.